openshift/release/knative-eventing-v0.13.0.yaml

```diff
@@ -2262,7 +2262,7 @@ spec:
     spec:
       serviceAccountName: eventing-controller
       containers:
-      - name: eventing-controller
+      - name: channel-broker-controller
```

Comment:
you "change" this file, by running the below, right ?

```
make RELEASE=v0.13.0 generate-release 
```

At least I do not get a _diff_ on the file

Reply:
kind of...